packages/terra-collapsible-menu-view/src/terra-dev-site/test/collapsible-menu-view/PrePopulatedCollapsedMenuItems.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import IconSend from 'terra-icon/lib/icon/IconSend';
+import CollapsibleMenuView from '../../../CollapsibleMenuView';
+
+const CollapsibleMenuViewDemo = () => (
+  <CollapsibleMenuView alwaysCollapsedMenuItems={[
+    <CollapsibleMenuView.Divider key="CollapsedDivider" />,
```

Comment:
Here is the double-divider @mjhenkes mentioned here https://github.com/cerner/terra-framework/pull/1347#issuecomment-759580441- looks out of place since it is the first item, move it lower or put 2 items above and 2 items below.

Reply:
```suggestion
```